portfolio/src/main/webapp/script-home.js

```diff
@@ -139,8 +144,145 @@ function deleteComments() {
  * Creates a map using the Google Maps API.
  */
 function initMap() {
-  const map = new google.maps.Map(document.getElementById('map'), {
+  // A custom color scheme of the map + basic properties for initialization
+  // (like initial center of map + zoom level)
+  const mapOptions = {
     center: { lat: 37.422, lng: -122.084 },
-    zoom: 9
+    zoom: 9,
+    styles: [
+      {
+        "featureType": "road",
+        "stylers": [
+          {
+            "color": "#b31b1b"
+          }
+        ]
+      },
+      {
+        "featureType": "road.arterial",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#ffffff"
+          }
+        ]
+      },
+      {
+        "featureType": "road.highway",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#c0c0c0"
+          }
+        ]
+      },
+      {
+        "featureType": "road.local",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#b31b1b"
+          }
+        ]
+      },
+      {
+        "featureType": "water",
+        "elementType": "geometry.fill",
+        "stylers": [
+          {
+            "color": "#c0c0c0"
+          }
+        ]
+      },
+      {
+        "featureType": "water",
+        "elementType": "labels.text.fill",
+        "stylers": [
+          {
+            "color": "#b31b1b"
+          }
+        ]
+      }
+    ]
+  };
+  const markers = [
+    {
+      name: 'Collegetown Bagels',
+      location: new google.maps.LatLng(42.442262, -76.484958),
+      content: 'One of my favorite places to break my fast in the' +
+          ' mornings, highly recommend!'
+    },
+    {
+      name: 'Cozumel, Mexico',
+      location: new google.maps.LatLng(20.4230, -86.9223),
+      content: 'Went here for the one and only cruise I have ever been' +
+          ' on, I don\'t recommend cruises for the faint of heart.'
+    },
+    {
+      name: 'Haiti',
+      location: new google.maps.LatLng(18.9712, -72.2852),
+      content: 'The land of my peoples 😃'
+    },
+    {
+      name: 'Bermuda Triangle',
+      location: new google.maps.LatLng(25.52, -70.35),
+      content: 'Spooky!'
+    },
+    {
+      name: 'Delaware',
+      location: new google.maps.LatLng(39.1852, -75.5244),
+      content: 'Also known as the first state, and that\'s all ' +
+          'it is known for!'
+    },
+    {
+      name: 'Anaheim, CA',
+      location: new google.maps.LatLng(33.8366, -117.9143),
+      content: 'Went here for a \"field trip\" once; Disney is pretty ' +
+          'cool + In-N-Out is overrated'
+    },
+    {
+      name: 'Kansas City, MO',
+      location: new google.maps.LatLng(39.0997, -94.5786),
+      content: 'Another \"field trip\", the food and vibes here were ' +
+          'amazing'
+    },
+  ];
+  // Constructs the polygon outlining the Bermuda Triangle
+  const bermudaTriangle = new google.maps.Polygon({
```

Comment:
same with bermuda triangle. in fact, if you want, you can create a js file called "map_constants.js" and dump these constants there. 

I forget whether javascript file names should have underscores or dashes